sentry-core/src/main/java/io/sentry/core/Hub.java

```diff
@@ -148,13 +149,59 @@ public boolean isEnabled() {
       } catch (Exception e) {
         options
             .getLogger()
-            .log(SentryLevel.ERROR, "Error while capturing message: " + throwable.getMessage(), e);
+            .log(
+                SentryLevel.ERROR, "Error while capturing exception: " + throwable.getMessage(), e);
       }
     }
     this.lastEventId = sentryId;
     return sentryId;
   }
 
+  @Override
+  public void startSession() {
+    StackItem item = this.stack.peek();
+    if (item != null) {
+      Scope.SessionPair pair = item.scope.startSession();
+      try {
+        item.client.captureSession(pair.getCurrent());
+      } catch (IOException e) {
+        options
+            .getLogger()
+            .log(SentryLevel.ERROR, "Error while capturing session: " + e.getMessage(), e);
+      }
+      if (pair.getPrevious() != null) {
+        try {
+          item.client.captureSession(pair.getPrevious());
```

Comment:
We could have `captureSession(Iterator<Session> sessions)` and capture both sessions at once.